swagger.yaml

```diff
@@ -231,6 +231,24 @@ paths:
           description: Unexpected error
           schema:
             $ref: "#/definitions/Error"
+    delete:
+      summary: Delete the current user account.
+      security:
+        - Bearer: [ ]
+      tags:
+        - Users
+      operationId: deleteUser
```

Comment:
Can we rename 'deleteUser' to 'deleteCurrentUser' to differentiate it from user deletion by admin?